database/src/main/postgres/runs/V0.3.0.2__patch_partitioning_parent.sql

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION runs.patch_partitioning_parent(
```

Comment:
I am not sure about what the function does and is supposed to do. I thought we should check that partitioning and parent partitioning exists and then simply add the relationship (if not already there) using _add_to_parent_flows.sql ...